### PR TITLE
doc: add Win 2022 support and update OS details

### PR DESCRIPTION
**Pull request by savijatv**
Please review the workflow files for other operating systems as well. Shouldn't this be the daisy-workflow files in the compute-image-import repo?

### CODE REVIEW

**Comment by yswe** [comment]
@savijatv, I agree here with you, it looks to me that we have to update all the paths to `daisy_workflows/image_import/...` Thank you!

File: docs/image-import.md [hunk 1]
```diff
@@ -263,16 +275,16 @@ import. Below is a list of known compatibility requirements and issues:
...
 | Multiple Disks | Warning  | Image import cannot directly handle multiple disk scenarios. Additional disks must be imported and attached separately. |
 | Powershell (Windows) | Warning | Warn if Powershell Version < 3. Powershell versions older than 3.0 can cause issues with GCE startup and shutdown scripts. |
 
 ### Linux
 
 | Name | Severity | Description |
 |---|---|---|
-| OS Version | Required | We support the following OS versions: RHEL/CentOS/OEL 6 or 7; Debian 9; Ubuntu 14.04 or 16.04. |
+| OS Version | Required | We support the following OS versions: RHEL 6, 7, 8, or 9; CentOS/OEL 7; Debian 8, 9, 10, or 11; Rocky Linux 8; SLES 12 or 15; Ubuntu 14.04, 16.04 or 18.04. See [supported operating systems](https://cloud.google.com/compute/docs/images/os-details#import)|
 | OS Disk | Required | The disk containing the OS must be bootable. The disk must be MBR and have GRUB installed. |
```
Comment (MahmoudNada0) [inline docs/image-import.md:288]: Here also we should clarify that we support GPT & GRUB2 as well.

**Review by yswe — APPROVED** [review]

File: docs/image-import.md [hunk 2]
```diff
@@ -173,6 +173,12 @@ Where, `YOUR_IMPORTED_IMAGE` is the GCE image that was created in step 2. The `s
     <td>/daisy/windows/translate_windows_2016.wf.json
     </td>
   </tr>
+  <tr>
+    <td>Windows Server 2008 R2
+    </td>
+    <td>/daisy/windows/translate_windows_2008_r2.wf.json
```
[the remark this answers is not in the document]
Comment (MahmoudNada0) [reply docs/image-import.md:179]: we still need to update the paths for the workflows files to start with "daisy_workflows/image_import/...".

File: docs/image-import.md [hunk 3]
```diff
@@ -156,9 +156,9 @@ Where, `YOUR_IMPORTED_IMAGE` is the GCE image that was created in step 2. The `s
     </td>
   </tr>
   <tr>
-    <td>Windows Server 2008 R2
+    <td>Windows Server 2022
     </td>
-    <td>/daisy/windows/translate_windows_2008_r2.wf.json
+    <td>/daisy_workflows/image_import/windows/translate_windows_2022.wf.json
```
[the remark this answers is not in the document]
Comment (MahmoudNada0) [reply docs/image-import.md:161]: do we have windows 2019 in this table?
also, w didn't we keep the Windows versions sorted?

**Pull request merged**